x-pack/plugins/synthetics/public/apps/synthetics/render_app.tsx

```diff
@@ -62,11 +62,11 @@ export function renderApp(
         links: [
           {
             linkType: 'documentation',
-            href: `${docLinks.links.observability.monitorUptime}`, // TODO: Include synthetics link
+            href: `${docLinks.links.observability.monitorUptimeSynthetics}`,
           },
           {
             linkType: 'discuss',
-            href: 'https://discuss.elastic.co/c/uptime', // TODO: Include synthetics link
+            href: 'https://discuss.elastic.co/c/uptime', // Redirects to https://discuss.elastic.co/c/observability/synthetics/75
```

Comment:
Just curious.. is there any reason not to point to the synthetics link instead of triggering a redirect?

Reply:
I am not confident how static is `https://discuss.elastic.co/c/observability/synthetics/75` (seeing `75` in it), so I left it as is. However, I am open to suggestions.